apps/core-app/src/components/customFields/nestedArrayInput.tsx

```diff
@@ -0,0 +1,12 @@
+import { Buildable, WrappedInput, Field } from '@daohaus/ui';
+import React from 'react';
+import { RegisterOptions } from 'react-hook-form';
+
+export const NestedArray = (props: Buildable<Field>) => {
+  const newRules: RegisterOptions = {
+    ...props.rules,
+    setValueAs: (val) => [val],
+    // validate: (val) => props?.rules?.validate? props.rules?.validate(val[0]) : true,
```

Comment:
Keeping this here for now until we think of a better way of handling nested validation.